### DISCUSSION TITLE
Node Red and Espaltherma

### DISCUSSION THREAD

**Discussion by alexdelli** [raomin/ESPAltherma#172] [Q&A] [answered]
Hi all
could you give me an help about send data Espaltherma from mqtt to influxdb via NodeRed?

I have the connection to Espaltherma and to influxdb, but I miss to retrieve data

![image](https://user-images.githubusercontent.com/6674409/196633749-baf87149-f19f-48f5-9816-729ad4d4a366.png)

**Comment by raomin** [comment, marked as answer]
Hi @alexdelli 
I guess this would help https://github.com/raomin/ESPAltherma#im-using-openhab-or-others-can-i-get-the-values-in-separated-mqtt-topics

**Reply by alexdelli** [reply]
Wow, it works, thank you
